src/libs/Navigation/AppNavigator/Navigators/OnboardingModalNavigator.tsx

```diff
@@ -29,6 +48,9 @@ function OnboardingModalNavigator() {
 
     useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.ESCAPE, handleOuterClick, {shouldBubble: true});
 
+    if (!hasCompletedGuidedSetupFlow) {
+        return;
```

Comment:
```suggestion
    if (hasCompletedGuidedSetupFlow) {
        return null;
```

@skyweb331 
Why did you change to negation here? 

Why is `null` omitted here? Returning `undefined` is not allowed for React components. You should explicitly return `null`.

Reply:
@c3024 Done. Thanks for your great point out.